public/pages/CreateMonitor/components/VisualGraph/utils/helpers.js

```diff
@@ -111,13 +116,13 @@ export function getDataFromResponse(response, fieldName, monitorType) {
   }
 }
 
-// Function for aggregation type monitors to get Map of data. 
-// The current response gives a large number of data aggregated in buckets, and this function returns the top n results with highest count of data points. 
-// The number n is based on the constant BAY_KEY_COUNT.  
+// Function for aggregation type monitors to get Map of data.
+// The current response gives a large number of data aggregated in buckets, and this function returns the top n results with highest count of data points.
+// The number n is based on the constant BAY_KEY_COUNT.
 export function getMapDataFromResponse(response, fieldName, groupByFields) {
   if (!response) return [];
   const buckets = _.get(response, 'aggregations.composite_agg.buckets', []);
-  let allData = new Map();
+  const allData = new Map();
```

Comment:
Wont const make it not possible to add new values to it?

Reply:
My understanding is that `const` prevents reassigning the variable's value, but does not prevent using the value's own methods (in this case `set`) to mutate it.